portfolio/src/main/webapp/script.js

```diff
@@ -43,10 +43,20 @@ function randomizeRecipe() {
   imageContainer.appendChild(imgElement);
 }
 
-async function getMessage() {
-  console.log('Fetching a message.');
+async function getComments() {
+  console.log('Fetching the comments.');
  
   const response = await fetch('/data');
-  const message = await response.json();
-  document.getElementById('message').innerHTML = message;
+  const comments = await response.json();
+
+  const commentsEl = document.getElementById('comments');
```

Comment:
nit: `commentsEl` --> `commentsElement` for clarity? (Should be able to easily tell what it is by variable name)